core/src/program/blocks/span_block.rs

```diff
@@ -659,25 +658,6 @@ mod tests {
         assert_eq!(hasher::hash_elements(&all_groups[..10]), hash);
     }
 
-    #[test]
-    fn batch_ops_with_decorator() {
-        let ops = vec![
-            Operation::Push(Felt::ONE),
-            Operation::Add,
-            Operation::Debug(DebugOptions::All),
```

Comment:
I'd prefer not to remove this test. Maybe just replace the `Debug` decorator with `Advice` decorator?

Reply:
Fixed.